src/app/lib/globalStyles.test.js

```diff
@@ -6,7 +6,8 @@ describe('globalStyles', () => {
 
     await import('./globalStyles');
 
-    const injectGlobalArgs = styledComponents.injectGlobal.mock.calls[0];
+    // For explaination on the dot nation ignore see https://github.com/bbc/simorgh/pull/601/files#r214874027
+    const injectGlobalArgs = styledComponents['injectGlobal'].mock.calls[0]; // eslint-disable-line dot-notation
```

Comment:
I could not find the link to the explanation. What is the relationship between eslint-disabling this line and fixing the travis error?